google/cloud/spanner/admin/integration_tests/database_admin_integration_test.cc

```diff
@@ -486,35 +488,39 @@ TEST_F(DatabaseAdminClientTest, CreateWithEncryptionKey) {
   auto database = client_.CreateDatabase(creq).get();
   ASSERT_STATUS_OK(database);
   EXPECT_EQ(database->name(), database_.FullName());
-  EXPECT_TRUE(database->has_encryption_config());
-  if (database->has_encryption_config()) {
-    EXPECT_EQ(database->encryption_config().kms_key_name(),
-              encryption_key.FullName());
-  }
+  if (emulator_) {
+    EXPECT_FALSE(database->has_encryption_config());
+  } else {
+    EXPECT_TRUE(database->has_encryption_config());
+    if (database->has_encryption_config()) {
+      EXPECT_EQ(database->encryption_config().kms_key_name(),
+                encryption_key.FullName());
+    }
 
-  auto get_result = client_.GetDatabase(database_.FullName());
-  ASSERT_STATUS_OK(get_result);
-  EXPECT_EQ(database->name(), get_result->name());
-  EXPECT_TRUE(get_result->has_encryption_config());
-  if (get_result->has_encryption_config()) {
-    EXPECT_EQ(get_result->encryption_config().kms_key_name(),
-              encryption_key.FullName());
-  }
+    auto get_result = client_.GetDatabase(database_.FullName());
+    ASSERT_STATUS_OK(get_result);
+    EXPECT_EQ(database->name(), get_result->name());
+    EXPECT_TRUE(get_result->has_encryption_config());
+    if (get_result->has_encryption_config()) {
+      EXPECT_EQ(get_result->encryption_config().kms_key_name(),
+                encryption_key.FullName());
+    }
 
-  // Verify that encryption config is returned via ListDatabases().
-  auto list_db = [&] {
-    for (auto const& db :
-         client_.ListDatabases(database_.instance().FullName())) {
-      if (db && db->name() == database_.FullName()) return db;
+    // Verify that encryption config is returned via ListDatabases().
+    auto list_db = [&] {
+      for (auto const& db :
+           client_.ListDatabases(database_.instance().FullName())) {
+        if (db && db->name() == database_.FullName()) return db;
+      }
+      return StatusOr<google::spanner::admin::database::v1::Database>{
+          Status{StatusCode::kNotFound, "disappeared"}};
+    }();
+    ASSERT_THAT(list_db, IsOk());
```

Comment:
No action needed: there may be a clever way to use `IsOkAndHolds()` here.

Reply:
Ack.  This was undoubtedly written before we had the `AndHolds` variety.  Perhaps we should add a "cleanup" issue to revisit such old examples, but I imagine we'd never think it was worth the effort.